Valery_Kashentsev/src/components/Messenger.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+import { MessagesList } from 'components/MessagesList';
+import { MessageForm } from 'components/MessageForm';
+
+export class Messenger extends Component {
+  state = {
+    messages: [
+      { text: 'Привет всем! Это первое сообщение', author: 'Dmitry' }
+    ],
+  };
+
+  interval = null;
+
+  componentDidUpdate() {
+    let author = this.state.messages[this.state.messages.length - 1].author;
+    if (author !== 'Bot') {
+      setTimeout(() => {
+        this.setState({
+          messages: this.state.messages.concat([{ text: 'Привет, ' + author + '! Бот на связи! Я не понимаю вас!', author: 'Bot' }]),
+        });
+      }, 1000);
+    }
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.interval);
+  }
+
+  handleMessageSend = (message) => {
+    this.setState({
+      messages: this.state.messages.concat(message),
```

Comment:
concat принимает аргументом массив
```suggestion
      messages: this.state.messages.concat([message]),
```